prow/caching.md

```diff
@@ -0,0 +1,50 @@
+# Caching dependencies in Prow
+
+Caching dependencies in prow works only when the [Pod Utilities][0] are enabled. A
+valid GCS Bucket has to be configured via `gcs_configuration` either in the Prow
```

Comment:
Please do a `s/GCS/Object Storage/g`, we also support S3 :)

Reply:
Done :+1: